database/migrations/default/1744126149551_noncr3_drop_date_hour_cols/up.sql

```diff
@@ -0,0 +1,244 @@
+--
+-- add a timestamptz column
+--
+alter table atd_apd_blueform add column case_timestamp timestamptz;
+comment on column atd_apd_blueform.case_timestamp is 'The timestamp at which the APD case occurred';
```

Comment:
could maybe be helpful to add `created_at` and `created_by` columns to the blueform table?

Reply:
thanks for that suggestion: https://github.com/cityofaustin/vision-zero/pull/1761/commits/dea6d4d41a6a11142c72e2410533d6d0aa9de23b